src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx

```diff
@@ -151,7 +151,7 @@ export const PDFPrintFeaturesGroup: React.FunctionComponent<{
                             }}
                         >
                             <ApiCheckbox
-                                english="Full Bleed"
+                                label="Full Bleed"
```

Comment:
I manually tested some but not all of the ApiCheckboxes that got touched. This one, for example, I wasn't sure how to enable